src/frontend/components/_atoms/graphical-editor/block-output-debug-badges.tsx

```
@@ -2,6 +2,7 @@ import { useDebugCompositeKey } from '../../../hooks/use-debug-composite-key'
 import { useIsDebuggerVisible } from '../../../hooks/use-debug-value'
 import { useIsGraphicalEditorActive } from '../../_features/[workspace]/editor/graphical/active-context'
 import { DebugValueBadge } from './debug-value-badge'
+import { blockOutputVariables } from '../../../utils/graphical/in-out-pin-rules'
```

ESLint

[error] Run autofix to sort these imports!

(simple-import-sort/imports)